ui/plugins/eu.esdihumboldt.hale.ui.functions.groovy/src/eu/esdihumboldt/hale/ui/functions/groovy/internal/JoinTypeStructureTray.java

```diff
@@ -40,6 +40,7 @@
  */
 public class JoinTypeStructureTray extends TypeStructureTray {
 
+	private static boolean isMerge = false;
```

Comment:
In the join type structure tray there is no variable needed, we know that because it is for the Join, it is not a Merge.